appstore/core/templates/account/login.html

```diff
@@ -1,4 +1,4 @@
-{% extends "account/base.html" %}
+{% extends "base.html" %}
```

Comment:
Was templates/accounts a directory created by us at some point and you are just renaming and restructuring for better?

Reply:
This was part of the Django 3 PR. There was an update to the `allauth` project that changed our references.

Reply:
https://github.com/pennersr/django-allauth/blob/master/ChangeLog.rst#backwards-incompatible-changes-1